core/presentation/src/commonMain/kotlin/zed/rainxch/core/presentation/components/RepositoryCard.kt

```
@@ -515,3 +513,43 @@ fun RepositoryCardPreview() {
+                fontWeight = FontWeight.Medium,
+                maxLines = 1,
+                softWrap = false,
+                overflow = TextOverflow.Ellipsis,
+            )
+        }
+    }
+}
+
+private fun formatCount(count: Long): String =
+    when {
+        count >= 1_000_000 -> String.format("%.1fM", count / 1_000_000.0)
```

detekt (1.23.8)

[warning] String.format("%.1fM", count / 1_000_000.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)